doc/source/tune/user-guide.rst

```diff
@@ -358,6 +358,9 @@ Tune provides a wrapper function ``tune.with_parameters()`` that allows you to b
 Objects passed with this wrapper will be stored on the Ray object store and will be automatically fetched
 and passed to your trainable as a parameter.
 
+.. tip:: If the objects are small in size or already exist in the Object Store, there's no need to use ``tune.with_parameters()``. You can use partials or pass in directly to ``config`` instead.
```

Comment:
Can you link to https://docs.ray.io/en/master/walkthrough.html#objects-in-ray for the object store?

Reply:
Also, I would link to 'partials' documentation; a lot of people using Tune are really basic programmers

Reply:
Good suggestions. I added both.